frontend/src/ramp.pug

```diff
@@ -1,13 +1,13 @@
 .ramp
-  template(v-if="tframe === 'commit'")
+  template(v-if="tframe === 'commit' || zoomfromramp")
```

Comment:
Is it really needed to have another `zoomfromramp` field? Can't we just use tframe? I.e. when we call openTabZoom from the ramp, we pass the tframe to be `commit`.

Reply:
Need, otherwise in zoom will have a list of commit contributed on the same day which is not true.

Reply:
> Instead of showing week of 2018-06-04 [#11] Implement logger as the title, standardising it with normal ramps with granularity commits (i.e. displays the true commit date) would be a better behaviour.

If the implementation follows this, then `zoomfromramp` won't be needed.